src/Term/TermTypes.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Wikibase\DataModel\Term;
+
+/**
+ * Term types used on Items and Properties
+ *
+ * @since 9.3
+ *
+ * @license GPL-2.0-or-later
+ */
+interface TermTypes {
+
+	public const TYPE_LABEL = 'label';
+	public const TYPE_DESCRIPTION = 'description';
+	public const TYPE_ALIAS = 'alias';
```

Comment:
Maybe having one for aliases would be good too? Just thinking out loud.

Reply:
The scary thing is that some services (e.g. `TermBuffer` and by association `PrefetchingTermLookup` and friends) expect to be passed `'alias'` not `'aliases'` to fetch all aliases. That can probably be solved by documenting it properly, but my original motivation was actually to have these constants to avoid exactly this mix-up.